src/django_components/component.py

```diff
@@ -589,7 +631,18 @@ def on_render_after(self, context: Context, template: Template, content: str) ->
     # MISC
     # #####################################
 
-    _class_hash: ClassVar[str]
+    class_id: ClassVar[str]
+    """
+    Unique ID of the component class, e.g. `MyComponent_ab01f2`.
+
+    This is derived from the component class' module import path, e.g. `path.to.my.MyComponent`.
+    """
+
+    # TODO_V1 - Remove this in v1
+    @property
+    def _class_hash(self) -> str:
```

Comment:
I've kept `_class_hash` in case someone relied on it, but marked it as deprecated.